libcrux-ml-dsa/src/simd/traits.rs

```diff
@@ -16,6 +19,140 @@ pub const INVERSE_OF_MODULUS_MOD_MONTGOMERY_R: u64 = 58_728_449;
 /// We use 'fer' as a shorthand for this type.
 pub(crate) type FieldElementTimesMontgomeryR = i32;
 
+type SIMDContent = [i32; COEFFICIENTS_IN_SIMD_UNIT];
+
+#[cfg(not(eurydice))]
+#[hax_lib::attributes]
+pub(crate) trait Repr: Copy + Clone {
+    #[requires(true)]
+    fn repr(&self) -> SIMDContent;
+}
+
+#[cfg(hax)]
+pub(crate) fn int_is_i32(i: hax_lib::Int) -> bool {
```

Comment:
Why not put all the proof pieces into a common module here instead of polluting the main code? Then you also need only one `cfg` guard.

Reply:
yes, good idea.

Reply:
I did the change, but had to tinker a little bit to avoid creating a recursive bundle. In a future PR, we may consider moving Repr also into the specs module.